readthedocs/builds/models.py

```diff
@@ -698,6 +720,15 @@ class Build(models.Model):
         null=True,
         blank=True,
     )
+    readthedocs_yaml_data = models.ForeignKey(
```

Comment:
Are we OK with this name? Other alternatives are:

- `readthedocs_yaml` (no `_data` at the end)
- `yaml_config`
- `config_yaml`
- `config_file`

cc @ericholscher what do you think?

Reply:
Presumably we should call it `build_config` if that's the model name we're going with. I think we probably want `yaml_config` for both?